guides/doc-Planning_Guide/topics/Provisioning_Concepts.adoc

```diff
@@ -62,40 +62,109 @@ For more information, see xref:figu-Satellite_Topology_with_Isolated_Capsule[].
+
+* Ensure that the subnet that functions as the host's provisioning interface has the following requirements:
++
+** a DHCP {SmartProxy}
+** a HTTP Boot {SmartProxy}
+** a TFTP {SmartProxy}
+** a Templates {SmartProxy}
+
+* Update the `grub2-efi` package to the latest version and execute the installer to copy the recent bootloader from `/boot` into `/var/lib/tftpboot` directory:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-install-project} grub2-efi
+# {foreman-installer}
+----
+
+==== HTTP Booting Requirements with unmanaged DHCP
+To provision machines through HTTP booting without managed DHCP ensure that you meet the following requirements:
+
 .Client requirements
 
+* HTTP UEFI Boot URL must be set to one of:
+** `http://{smartproxy.example.com}:8000
+** `https://{smartproxy.example.com}:{smartproxy_port}
+* Ensure that your client has access to the DHCP and DNS servers.
+* Ensure that your client has access to the HTTP UEFI Boot {SmartProxy}.
 * Ensure that all the network-based firewalls are configured to allow clients on the subnet to access the {SmartProxy}.
 For more information, see xref:figu-Satellite_Topology_with_Isolated_Capsule[].
 
-* Ensure that your client has access to the DHCP and TFTP servers.
+.Network requirements
+
+* An unmanaged DHCP server available for clients.
+* An unmanaged DNS server available for clients. In case DNS is not available, use IP address to configure clients.
 
 .{Project} requirements
 
+Although TFTP protocol is not used for HTTP UEFI Booting, {Project} use TFTP {SmartProxy} API to deploy bootloader configuration.
+
 * Ensure that both {ProjectServer} and {SmartProxy} have DNS configured and are able to resolve provisioned host names.
 
 * Ensure that the UDP ports 67 and 68 are accessible by the client so that the client can send and receive a DHCP request and offer.
 
-* Ensure that the TCP port 8000 is open for the client to download files and Kickstart templates from the {SmartProxy}.
+* Ensure that the TCP port 8000 is open for the client to download bootloader and Kickstart templates from the {SmartProxy}.
 
-* Ensure that the host provisioning interface subnet has a DHCP {SmartProxy} set.
+* Ensure that the TCP port {smartproxy_port} is open for the client to download the bootloader from the {SmartProxy} via HTTPS protocol.
 
 * Ensure that the host provisioning interface subnet has a HTTP Boot {SmartProxy} set.
 
+* Ensure that the host provisioning interface subnet has a TFTP {SmartProxy} set.
+
 * Ensure that the host provisioning interface subnet has a Templates {SmartProxy} set.
 
-* Update the `grub2-efi` package to the latest version:
+* Update the `grub2-efi` package to the latest version and execute the installer to copy the recent bootloader from the `/boot` directory into the `/var/lib/tftpboot` directory:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # {package-install-project} grub2-efi
+# {foreman-installer}
+----
```

Comment:
Because there are so many deletions, I can't apply the change here. But please do the same.